### PR TITLE
feat: attachment dnd

### PR DESCRIPTION
**Pull request by zholadev**
Drag and drop for files

### CODE REVIEW

**Comment by tea-artist** [comment]
Hi @zholadev can you provide detailed instructions (screenshot or video) for using this feature?

**Comment by zholadev** [comment]
Hi

пн, 2 сент. 2024 г. в 18:11, Bieber ***@***.***>:

**Comment by zholadev** [comment]
Uploading zapis-ekrana-2024-09-02-v-181409_1qc8ttIV.mp4…

**Comment by zholadev** [comment]
I sent the video to your email. Did you watch it?

File: apps/nextjs-app/.env.development [hunk 1]
```diff
@@ -22,7 +22,7 @@ STORAGE_PREFIX=http://127.0.0.1:3000
 
 # DATABASE_URL
 # @see https://www.prisma.io/docs/reference/database-reference/connection-urls#examples
-PRISMA_DATABASE_URL=postgresql://teable:teable@127.0.0.1:5432/teable?schema=public&statement_cache_size=1
```
Comment (tea-artist) [inline apps/nextjs-app/.env.development]: please remove this change

File: packages/sdk/src/components/editor/attachment/upload-attachment/UploadAttachment.tsx [hunk 2]
```diff
@@ -134,6 +160,30 @@ export const UploadAttachment = (props: IUploadAttachment) => {
 
   const uploadingFilesList = map(uploadingFiles, (value, key) => ({ id: key, ...value }));
 
+  const handleDragEnd = (event: DragEndEvent) => {
+    const { active, over } = event;
+
+    // Проверьте, что 'over' не undefined
```
Comment (tea-artist) [inline packages/sdk/src/components/editor/attachment/upload-attachment/UploadAttachment.tsx]: Please use English comment

File: packages/sdk/src/components/editor/attachment/upload-attachment/FIleImageItem.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        </div>
+        <span className="mt-1 w-full truncate text-center" title={attachment.name}>
+          {attachment.name}
+        </span>
+      </li>
+    </div>
+  );
+}
+
+export default FIleImageItem;
```
Comment (tea-artist) [inline packages/sdk/src/components/editor/attachment/upload-attachment/FIleImageItem.tsx]: Spelling error, There is also a problem with the file name.
Directly changing the case may cause git to fail to track changes. You can rename it to AttachmentItem or FileItem.

**Comment by tea-artist** [comment]
Please make sure it can run on the mobile web page.

**Comment by zholadev** [comment]
mobile adaptation works

File: packages/sdk/src/components/editor/attachment/upload-attachment/DragAndCopy.tsx
Comment (boris-w) [inline packages/sdk/src/components/editor/attachment/upload-attachment/DragAndCopy.tsx:1]: This file has deleted.

**Review by boris-w — APPROVED** [review]

**Review by tea-artist — APPROVED** [review]

**Comment by tea-artist** [comment]
Merge on Monday 🍺

**Pull request merged**